internal/ethapi/api.go

```diff
@@ -972,7 +972,7 @@ func (e *revertError) ErrorData() interface{} {
 // Note, this function doesn't make and changes in the state/blockchain and is
 // useful to execute and retrieve values.
 func (s *PublicBlockChainAPI) Call(ctx context.Context, args TransactionArgs, blockNrOrHash rpc.BlockNumberOrHash, overrides *StateOverride) (hexutil.Bytes, error) {
-	result, err := DoCall(ctx, s.b, args, blockNrOrHash, overrides, 5*time.Second, s.b.RPCGasCap())
+	result, err := DoCall(ctx, s.b, args, blockNrOrHash, overrides, s.b.RPCCallTimeout(), s.b.RPCGasCap())
```

Comment:
You can also put this timeout in `DoEstimate`

Reply:
Assuming you mean `DoEstimateGas`? Wouldn't that be a regression for anyone using `eth_estimateGas` before sending XL transactions? Currently `estimateGas` would not time out while running the EVM call. Adding this change here would enforce a new default limit.